README.md

```diff
@@ -76,15 +76,10 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://emekaemmanuel.github.io/Bookstore
-) 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## Project Presentation <a name="project-presentation"></a>
 
-- [Project Presentation video link](https://) 
-
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] Under the **Getting Started** section:
- I would advise you to keep only `Set up`, `Prerequisites`, `install`, `usage`, and Remove other sections like `Available Scripts`, `Learn more` etc ... **because they were not mentioned in the Table of Contents**

- [ ] NB: Normally your project was supposed to be invalidated due lack of coherence you made in the Readme. I would strongly advise you not to repeat these mistakes.